src/robot/htmldata/libdoc/libdoc.html

```diff
@@ -376,5 +378,15 @@ <h2 id="Keywords">Keywords</h2>
     </p>
 </script>
 
+
+<script type="text/javascript">
+    params = utils.parseQueryString(window.location.search.slice(1))
+    if ("tag" in params) {
+        $(document).ready(function() {
+          tagSearch(params["tag"])
+        })
+    }
```

Comment:
1. Shouldn't it be `util`, not `utils`?

2. Could this be move to the top of this file where the main logic is? Something like this:

    ```
    if ("tag" in params) {
        tagSearch(params["tag"]);
    else
        scrollToHash();
    ```

    Not sure is this possible (haven't looked at this code for ages) but it would feel better to have the main logic in same place.

3. Our JS style (which hasn't been explicitly documented anywhere) has been including `;` at the end of lines.

Reply:
1. yes.
2. yes but..

location for this code is definitely better suited to where you point  yes... And there's few things i need to work on so that current functionality work get broken.

3. yes boss.

Reply:
went thru 2. and if i understand the rfc correctly, something like this still works fine in my usecase: `?tag=plugin#Plugin%3A SeleniumTestability`  .. eg, in my case the link would be: https://rasjani.github.io/robotframework-seleniumtestability/index.html?tag=plugin#Plugin%3A%20SeleniumTestability